src/ncp/ncp_openthread.cpp

```diff
@@ -285,6 +286,9 @@ otError ControllerOpenThread::ApplyFeatureFlagList(const FeatureFlagList &aFeatu
 #if OTBR_ENABLE_DHCP6_PD
     otBorderRoutingDhcp6PdSetEnabled(mInstance, aFeatureFlagList.enable_dhcp6_pd());
 #endif
+#if OTBR_ENABLE_LINK_METRICS_TELEMETRY
+    otLinkMetricsManagerSetEnabled(mInstance, aFeatureFlagList.enable_link_metrics_mgr());
```

Comment:
Please make sure this API can be called (may be invoked with same flag value) multiple times and it has no issue.

Reply:
I've created a PR to test continuously switch the feature: https://github.com/openthread/openthread/pull/10035

I've run it locally and it works well.